app/models/application.rb

```diff
@@ -91,6 +91,7 @@ def votes_threshold_email
 
     event :submit do
       transition started: :submitted
+      transition rejected: :submitted
```

Comment:
Do we possibly need to add another state here? I'm wondering if there is a valid case where we reject someone and we don't want them to be able to apply again. I am not familiar enough with the application process to know in what situations someone would be rejected.

Reply:
Based on our current [Voting Members Policy](https://docs.google.com/document/d/12R7utXAiyCK55XEP8cPscUqu2CXAj-hL5jXE-MaJdgE/edit#heading=h.y6eimvxrwzvf), there are no cases where a person should be technologically prevented from applying again. There are a few cases where specific people would be simply rejected if they applied again.